Integrated a lot of feedback... (#233)
rocket-pool/RPIPs: RPIPs/RPIP-46.md

   3. `surplus_share`: this share of revenue is used to distribute revenue beyond that used for protocol operation (such as the shares above)
2. `reth_commission` SHALL be defined as the sum of all defined shares that have settings
3. `reth_share` SHALL be defined as `100% - reth_commission`
4. Distributions of revenue from borrowed ETH MUST respect the defined shares
   1. If shares change between claims, distributions MUST make an effort to account for the different values. For example, a distribution could use a duration-weighted average share. Approximations MAY be used where they significantly reduce complexity and/or costs.
   2. Legacy minipools are an exception and MAY continue to support earlier distribution methodologies 
5. `node_operator_commission_share` and `surplus_share`, MAY be updated by an address in the `allowlisted_controllers` array
   1. This functionality SHALL not be used without a separate pDAO vote to enable a controller and add it to the list

> 6. The `node_operator_commission_share_council_adder` setting SHALL only be set to values within 0% and `max_node_operator_commission_share_council_adder` (inclusive)

Reviewer: Feel like this needs a better name. no_share and no_share_council_mod?

### Inflation
1. Inflation settings SHALL be modified to retain inflation to the DAOs and eliminate inflation to NOs
   1. `rpl.inflation.interval.rate` SHALL be set to `1000040763630249500` (1.5% per year)
   2. Node Operators (`rocketClaimNode`) allocation SHALL be set to 0%
   3. pDAO (`rocketClaimDAO`) allocation SHALL be set to 95%
   4. oDAO (`rocketClaimTrustedNode`) allocation SHALL be set to 5%

### Surplus revenue share vote
Prior to the release of Saturn 1, a ranked-choice vote MUST be held to select a mechanism for surplus revenue share
1. The choices MAY include [RPIP-45: RPL Burn](RPIP-45.md), [RPIP-50: RPL LP](RPIP-50.md), using higher `voter_share`, and/or additional options
2. If higher `voter_share` is selected:
   1. The entirety of the ["Specification taking effect with Saturn 2"](#specification-taking-effect-with-saturn-2) section below SHALL be deleted
   2. `voter_share` SHALL become controllable by pDAO vote; however, it SHALL require a supermajority vote with at least 75% of the vote in support of any increase 
   3. `voter_share` MAY be updated by an address in the `allowlisted_controllers` array
3. The selected mechanism MAY be implemented in Saturn 1

>    1. If not, the revenue SHALL be held in reserve until the mechanism is implemented. In such a case, once the mechanism is implemented, the revenue SHALL be distributed in the same amount of time it took to build up, or faster.

Reviewer: Should we just delete this or make it a MAY? Seems mildly complicated and I'm not sure it's strictly necessary.

Author: Line 70?

Reviewer: Yes. Line 70.

A few details about the reasoning behind the spec:
- We provide for future automated controller contracts, but do not create any at this time. This is partly because we are fairly naive to the market, and partly in the interest of time to market for the next upgrade.
- The `node_operator_commission_share_council_adder` allows for much more rapidly tracking the market, especially when we first start UARS and may be quite far from the an appropriate `node_operator_commission_share + node_operator_commission_share_council_adder` value
  - Due to the low maximum for the adder, the pDAO would need to act to enable much growth in `node_operator_commission_share + node_operator_commission_share_council_adder`. For example, if the adder is at 1%, the pDAO could vote to set it to 0% and add 1% to `node_operator_commission_share`. This active pDAO participation ensures that this setting tracks closely to the will of the pDAO. 
- The [surplus revenue share vote](#surplus-revenue-share-vote) is intended to allow the main body of the tokenomics to move forward, while allowing more time to get information about our options here before choosing a path

## Security Considerations

> - `node_operator_commission_share_council_adder` is intended to be used in a particular way, but the security council may misuse it and increase it in other ways

Reviewer: ```suggestion
- `node_operator_commission_share_council_adder` is intended to be used in a particular way, but there is no on-chain mechanism to prevent the security council misusing it.
```

Reviewer: Ah, now doesn't track with your point below.
